example/methods_directory/physical/psp.json

```diff
@@ -23,6 +20,23 @@
                 "exchangeCorrelation": "/pb/qm/dft/ksdft/gga?functional=pbe",
```

Comment:
I think we should mirror here what we use right now and perhaps extend it to support path also. For example this could look like:
```json
{
    "exchangeCorrelation": {
        "approximation": "gga",
        "functional": "pbe",
        "path": "/pb/qm/dft/ksdft/gga/?functional=pbe"
    }
}
```
or alternatively we could also move the path to a `modelPath` property instead:
```json
{
    "exchangeCorrelation": {
        "approximation": "gga",
        "functional": "pbe"
    },
    "modelPath": "/pb/qm/dft/ksdft/gga/?functional=pbe"
}
```
What do you think?